website/docs/r/access_service_token.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "cloudflare"
+page_title: "Cloudflare: cloudflare_access_service_token"
+sidebar_current: "docs-cloudflare-resource-access-service-token"
+description: |-
+  Provides a Cloudflare Access Service Token resource.
+---
+
+# cloudflare_access_service_token
+
+Access Service Tokens are used for service-to-service communication
+when an application is behind Cloudflare Access.
+
+## Example Usage
+
+```hcl
+resource "cloudflare_access_service_token" "my_app" {
+  account_id = "d41d8cd98f00b204e9800998ecf8427e"
+  name       = "CI/CD app"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `account_id` - (Required) The ID of the account where the Access
+Service is being created.
+* `name` - (Required) Friendly name of the token's intent.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `client_id` - UUID client ID associated with the Service Token.
+* `client_secret` - A secret for interacting with Access protocols.
+
+## Import
+
+~> **Important:** If you are importing an Access Service Token you will
```

Comment:
🚨 this is really important 🚨 if you're intending on importing, this should be well known.

Reply:
Maybe just skip import if not useful?

Reply:
I wasn't sure if it would be completely useless as if you've already got a system that is using the token but you just want to manage everything in code; this is handy to have. The only caveat to remember is that you can't reference it in other resources. 

I wonder if on import, it should be set to a value that clearly states this incase someone is trying to reference it 🤔

Reply:
Better not to set any value, and maybe emit a warning.